### PR TITLE
Feat/form with interactions work 1873

### PR DESCRIPTION
**Pull request by M-Ryan92**

### CODE REVIEW

File: src/prefabs/structures/Box/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { component, PrefabComponentOption } from '@betty-blocks/component-sdk';
+import { PrefabReference } from '@betty-blocks/component-sdk/build/prefabs/types/component';
```
Comment (benjamin-rocks) [inline src/prefabs/structures/Box/index.ts:2]: oh nooos, this type should be exported/imported at the top level.

File: src/prefabs/structures/Box/options/advanced.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { toggle, variable } from '@betty-blocks/component-sdk';
+
+export const advanced = {
+  advancedSettings: toggle('Advanced settings', {
+    value: false,
+  }),
+  dataComponentAttribute: variable('Test attribute', {
+    value: ['Box'],
+    configuration: {
+      condition: {
```
Comment (benjamin-rocks) [inline src/prefabs/structures/Box/options/advanced.ts:10]: we have a `showIfTrue` helper for this ;)

File: src/prefabs/structures/Box/options/index.ts [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+      allowedInput: [
+        { name: 'None', value: 'none' },
+        { name: 'Left', value: 'flex-start' },
+        { name: 'Center', value: 'center' },
+        { name: 'Right', value: 'flex-end' },
+        { name: 'Justified', value: 'space-between' },
+      ],
+    },
+  }),
+  valignment: option('CUSTOM', {
```
Comment (benjamin-rocks) [inline src/prefabs/structures/Box/options/index.ts:27]: we have a buttongroup helper for this

File: src/prefabs/structures/Box/options/index.ts [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+    value: ['0rem', '0rem', '0rem', '0rem'],
+  }),
+  innerSpacing: option('SIZES', {
+    label: 'Inner space',
+    value: ['M', 'M', 'M', 'M'],
+  }),
+  positioningOptions: toggle('Show positioning options', {
+    value: false,
+  }),
+  position: option('CUSTOM', {
```
Comment (benjamin-rocks) [inline src/prefabs/structures/Box/options/index.ts:70]: we have a buttongroup helper for this

File: src/prefabs/structures/Box/options/index.ts [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+        value: true,
+      },
+    },
+  }),
+  top: sizes('Top position', {
+    value: '',
+    configuration: {
+      as: 'UNIT',
+      condition: {
+        type: 'SHOW',
```
Comment (benjamin-rocks) [inline src/prefabs/structures/Box/options/index.ts:96]: we have `showIfTrue` helper for this

File: src/prefabs/structures/Box/index.ts (hunk 1 above)
Comment (M-Ryan92) [reply src/prefabs/structures/Box/index.ts:2]: you know this happens at other places too already xD

**Review by benjamin-rocks — APPROVED** [review]

**Pull request merged**